gunpowder-api/src/main/kotlin/io/github/gunpowder/api/builders/TeleportRequest.kt

```diff
@@ -38,9 +39,7 @@ import java.util.function.Consumer
 
 interface TeleportRequest {
     val player: ServerPlayerEntity
-    val destination: Vec3d
-    val dimension: Identifier
-    val facing: Vec2f?
+    val destination: Location
```

Comment:
Make sure to open a PR for this on Gunpowder Teleport as this is a breaking API change.